tests/test_texmobject.py

```diff
@@ -164,3 +166,20 @@ def test_error_in_nested_context(capsys):
     stdout = str(capsys.readouterr().out)
     # validate useless context is not included
     assert r"\begin{frame}" not in stdout
+
+
+# TODO: ValueError: max() arg is an empty sequence error for some reason
+
+# def test_color_inheritance():
+#     """Test that Text and MarkupText correctly inherit colour from
+#     their parent class."""
+
+#     VMobject.set_default(color=RED)
+#     vmob = VMobject()
+#     tex = Tex("test_color_inheritance")
+#     mathtex = MathTex("test_color_inheritance")
+#     VMobject.set_default()
+
+
+#     assert tex.color == vmob.color
+#     assert mathtex.color == vmob.color
```

Comment:
```suggestion
def test_color_inheritance():
     """Test that Text and MarkupText correctly inherit colour from
     their parent class."""

     VMobject.set_default(color=RED)
     vmob = VMobject()
     tex = Tex(r"test\_color\_inheritance")
     mathtex = MathTex("test_color_inheritance")
     VMobject.set_default()


     assert tex.color == vmob.color
     assert mathtex.color == vmob.color
```
Underscores are reserved for subscripts in LaTeX.

Reply:
Ah good catch, I've turned this into a graphical test since I suspect it would have suffered from the same issue as the `text_mobject.py` test.